gatsby/content/blog/2022/02/2022-02-07-fosdem-recap.mdx

```diff
@@ -0,0 +1,54 @@
+---
+date: '2022-02-07'
+title: "Hosting FOSDEM 2022 on Matrix"
+categories:
+  - General
+author: Thib
+---
+
+Last year was the first time FOSDEM was hosted on Matrix, and it was generally a huge success - and so FOSDEM team trusted us again this year and we’re happy to say that it seems to have gone really well! This year’s FOSDEM was massive once again, featuring 654 speakers, 731 events, and 103 tracks.
```

Comment:
sorry, I missed a word in my original suggestion:
```suggestion
Last year was the first time FOSDEM was hosted on Matrix, and it was generally a huge success - and so the FOSDEM team trusted us again this year and we’re happy to say that it seems to have gone really well! This year’s FOSDEM was massive once again, featuring 654 speakers, 731 events, and 103 tracks.
```

Reply:
Nope, it was me. Thanks for watching my back!